core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -625,6 +625,29 @@ public void month() {
     assertEquals(integerValue(8), eval(expression));
   }
 
+  @Test
+  public void month_of_year() {
```

Comment:
Rest of the file uses `camelCase` for function names. Good to keep it consistent.